libraries/standard/mqtt/cbmc/proofs/MQTT_GetIncomingPacketTypeAndLength/MQTT_GetIncomingPacketTypeAndLength_harness.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Insert copyright notice
+ */
+
+/**
+ * @file MQTT_GetIncomingPacketTypeAndLength_harness.c
+ * @brief Implements the proof harness for MQTT_GetIncomingPacketTypeAndLength function.
+ */
+#include "mqtt.h"
+
+extern int32_t NetworkInterfaceReceiveStub( NetworkContext_t context,
+                                            void * pBuffer,
+                                            size_t bytesToRecv );
+
+void harness()
+{
+    NetworkContext_t networkContext;
+
```

Comment:
I'm surprised there is nothing more to allocation than just a pointer.  No backing struct?

Reply:
No because this is an application defined structure. I had comments there at first, I will add them back.